atramhasis/static/js/concept.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function() {
+    var searchUrl = window.location.search.substr(1);
+    const params = searchUrl.split('&');
+
+    if(params[1] === 'view_tree') {
```

Comment:
ik zou kijken of die ergens in de params zit, nie specifiek op die plaats; als er ooit params zouden bijkomen kan deze breken